### PR TITLE
Vectra AI - Analytics Templates for Detect

### PR DESCRIPTION
**Pull request by danymello**
Fixes #

## Proposed Changes

  - Set of Analytics Templates for Vectra AI Detect

### CODE REVIEW

**Comment by preetikr** [comment]
@v-jayakal - these would need to go to solutions and please review metadata of detections - thanks.
@shainw for detection review - thanks.

**Comment by v-admahe** [comment]
@danymello Could you please move the files in solution folder. You can take reference from [here](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Box)

**Comment by danymello** [comment]
@v-admahe, I created a new PR with all Vectra's content into Solutions: https://github.com/Azure/Azure-Sentinel/pull/2423 . It includes those Analytic rules + Data Connectors + Parser + Workbook.

**Comment by v-admahe** [comment]
@shainw We have reviewed metadata. Could you please take up further.

Thanks,
Aditi Maheshwari

**Comment by v-admahe** [comment]
@shainw We have reviewed metadata. Could you please take up further.

Thanks,
Aditi Maheshwari

**Comment by shainw** [comment]
> @shainw We have reviewed metadata. Could you please take up further.
> 
> Thanks,
> Aditi Maheshwari

I have approved, but I think many of these could be collapsed into 1 by using the [Customize alert details feature](https://docs.microsoft.com/en-us/azure/sentinel/customize-alert-details).  Example in the [Wiki](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride).

Please look at doing another PR to collapse these using the following, where we have the same query but a slightly different clause in each:

### "Vectra AI Detect - Account in the **{{Level}}** Quadrant" and  "Vectra AI Detect - Host in the **{{Level}}** Quadrant"
We can use a case() statement for this ```| where threat_score >= threat_score_min and certainty_score >= certainty_score_min``` to assign the variable and simply have 1 query that covers 4 detections.  

### For "Vectra Detection - **{{Category}}**"
we can simply assign the "cat" column to the variable.  

### For "Vectra AI Detect - **{{Category}}** Detection with High Severity"
Again we can separate the Severity match ```| where todecimal(LogSeverity) >= severity_threshold``` and Category match ```Category == "RECONNAISSANCE"```to another line away from the severity match and set the variable as above.

Basically, this moves us to 5 queries, but 20 detections.

**Review by shainw — APPROVED** [review]

**Comment by v-jayakal** [comment]
@danymello , please consider directions provided by @shainw for efficient solution for analytic rules and raise another PR. We are merging these for now.
